### ISSUE TITLE
Docs about new *byRole capabilities (+ Blog post?)

### ISSUE THREAD

**Issue by afontcu** [testing-library/dom-testing-library#283]
Hi! I guess the addition of https://github.com/testing-library/dom-testing-library/pull/280 should be documented.

I'm up for it, just wanted to clarify some doubts:

* Did the "level of recommendation" for `*byRole` change? ([source](https://testing-library.com/docs/guide-which-query)). I feel these queries got way more powerful now.

* Would a blog post be useful, too? Bot this and package scoping are quite big changes IMHO.

Actually, when writing these lines, I thought of something else. Right now, React/Vue/etc Testing Library users benefit from DTL additions, but they need to check DTL changelog to be aware of that. So, how would you feel about having like a "global" changelog for all the Testing Library ecosystem?

**Comment by kentcdodds** [comment]
I love all of the ideas 👍

**Comment by alexkrolick** [comment]
The changelogs are autogenerated, but a blog post on the docs would be great.

**Comment by afontcu** [comment]
Thanks both for the input!

@alexkrolick "changelog" is probably not the best way to put it. I think what I was trying to solve is that right now people need to look at several places to know about the news (DTL docs, VTL/RTL/etc docs, and so on).

Anyway, I'm closing this issue (I just noticed I opened it in the wrong repo 😂 damn) and opening a PR for blogpost, and another issue once I came up with a better suggestion for "unified" docs :)


Again, thanks!
